akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -418,6 +418,11 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
   def map[T](f: function.Function[Out, T]): javadsl.Flow[In, T, Mat] =
     new Flow(delegate.map(f.apply))
 
+  def mapTo[T](clazz: Class[T]): javadsl.Flow[In, T, Mat] = {
```

Comment:
methods will need docs

Reply:
Does this method really carry its weight if its implementation is *exactly* `map(Class.class::cast)`

Reply:
Yeah, I'm also not convinced, `filterType` is more obvious since there is no partial function in Javaland.